Server-Install.md

```diff
@@ -410,6 +410,9 @@ python reset-dbs.py \
 Or provide the credentials and values you configured.
 Check `python reset-dbs.py --help` for additional information.
 
+## Install Task Processing Unit (TPU)
+The Task Processing Unit (TPU) allows for batch-processing ingest, transform and export tasks. Refer to the [documentation in the TPU project](https://github.com/dswarm/task-processing-unit-for-dswarm).

```

Comment:
"Refer to the documentation in the TPU project" -> "Please have a look at the TPU documentation for further details on its usage and install instructions."